python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_network_services.schema.yml

```diff
@@ -1568,6 +1568,19 @@ $defs:
                       items:
                         type: str
                       description: Nodes is required to restrict configuration of BGP neighbors to certain nodes in the network.
+              aggregate_addresses:
+                type: list
+                items:
+                  type: dict
+                  $ref: "eos_cli_config_gen#/keys/router_bgp/keys/aggregate_addresses/items"
+                  keys:
+                    nodes:
+                      type: list
+                      description: Nodes is required to restrict configuration of aggregate_addresses to certain nodes in the network.
+                      required: true
```

Comment:
Not sure this should be required. The nodes in the other areas like bgp_peers are not.